lib/publiccloud/provider.pm

```diff
@@ -479,7 +479,12 @@ sub terraform_apply {
     record_info('TFM cmd', $cmd);
 
     script_retry($cmd, timeout => $terraform_timeout, delay => 3, retry => 6);
-    my $ret = script_run('terraform apply -no-color -input=false myplan', $terraform_timeout);
+
+    # Valid values according to documentation: TRACE, DEBUG, INFO, WARN, ERROR & OFF
+    # https://developer.hashicorp.com/terraform/internals/debugging
+    my $tf_log = get_var("TERRAFORM_LOG", "");
```

Comment:
Shouldn't default be `OFF`?

Reply:
> Shouldn't default be `OFF`?

I haven't tried it, but having the null string as default is like not setting `TF_LOG` at all.